concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -23,4 +23,4 @@ run:
       cf api "$CF_API"
       cf auth
       cf t -o "$CF_ORG" -s "$CF_SPACE"
-      cf push $CF_APP_NAME --strategy rolling
+      cf push $CF_APP_NAME --strategy rolling --hostname www -d $DOMAIN
```

Comment:
I think the domain is `london.cloudapps.digital` and it's the hostname we can change.

Reply:
I think the `--hostname` and `-d` options may not be available in the v7 CLI 🤔

Reply:
The [upgrade guide](https://docs.cloudfoundry.org/cf-cli/v7.html) suggests the route should be set in the manifest file instead.  For DGU, we have two manifests ([prod](https://github.com/alphagov/datagovuk_find/blob/master/production-manifest.yml) and [staging](https://github.com/alphagov/datagovuk_find/blob/master/staging-manifest.yml)).  If we switch to using the manifest files more (across all our apps), then we can probably eliminate most of the `cf <something>` lines from the Concourse task (and also stop the `cf scale` problem we had yesterday when the deployment failed).